docs/usage/docker.md

```diff
@@ -256,7 +256,7 @@ Renovate can authenticate with AWS ECR using AWS access key id & secret as the u
 
 #### Google Container Registry / Google Artifact Registry
 
-##### Using Application Default Credentials / Workload Identity
+##### Using Application Default Credentials / Workload Identity (Self-Hosted only)
```

Comment:
Why is this only working in self-hosted mode (same as for the your docs in `java.md`)?
True for Workload Identity but short-/ long-lived credentials should work anyway, or am I missing something?

Reply:
Those are separate headings though if I'm correct? Application Default Credentials also only work when running self-hosted since it's dependent on an environment variable/gcloud cli credentials/a metadata server.

Reply:
Correct. I mixed it up and actually meant to comment this regarding the headings in `java.md`.